cmd/heptio-authenticator-aws/token.go

```diff
@@ -67,4 +68,6 @@ func init() {
 	tokenCmd.Flags().StringP("role", "r", "", "Assume an IAM Role ARN before signing this token")
 	viper.BindPFlag("role", tokenCmd.Flags().Lookup("role"))
 	viper.BindEnv("role", "DEFAULT_ROLE")
+	tokenCmd.Flags().IntP("duration", "d", 15, "Seconds that the token is valid for.")
```

Comment:
I think this flag could be a [`DurationP`](https://godoc.org/github.com/spf13/pflag#FlagSet.DurationP) which should be a bit cleaner.